src/lib/composables/use-router-link.ts

```diff
@@ -27,13 +27,12 @@ export function isRouteActive(
 }
 
 export default function <T extends UseRouterLinkProps>(props: T) {
-  const hasLink = props.to !== undefined || props.href !== undefined;
-
+  const hasLink = [props.to, props.href].some((entry) => typeof entry !== "undefined");
```

Comment:
Why `typeof entry !== "undefined"` and not `entry !== undefined`?

Reply:
Good question. Because it's one of the safest ways to check if something is `undefined`, even though it became a bit less needed with ES5. Still, in some contexts `undefined` could be [be reassigned](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined#description).
So I just prefer the [`typeof` check](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined#typeof_operator_and_undefined). But we could also agree on [`void 0`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined#void_operator_and_undefined) if you prefer that?